src/java.base/share/classes/java/math/BigInteger.java

```diff
@@ -189,6 +189,7 @@ public class BigInteger extends Number implements Comparable<BigInteger> {
      * value.  Note that this implies that the BigInteger zero has a
      * zero-length mag array.
      */
+    @Stable
```

Comment:
`BigInteger.mag` gets assigned to a few times during computation in `largePrime()`:
https://github.com/openjdk/jdk/blob/db340e54f83cf7bf72abb94c9cf9cdac007ed38a/src/java.base/share/classes/java/math/BigInteger.java#L886-L887
https://github.com/openjdk/jdk/blob/db340e54f83cf7bf72abb94c9cf9cdac007ed38a/src/java.base/share/classes/java/math/BigInteger.java#L895-L898

Reply:
BigInteger is an object that is considered immutable, but its field mag is actually changed. This is incredible and should be fixed here. It is easy to make mistakes.

Reply:
> Please do _not_ fix the `mag` issue in this PR. I'll take care of it in a future PR.

Note that there is a troublesome thing here. There are many places in BigInteger that pass mag to MutableBigInteger